### PR TITLE
Adding Polish translation of the ultimate consent

### PR DESCRIPTION
**Pull request by mkoculak**
Hi,
I have used the ultimate consent in my studies (thanks for this useful resource!) and translated it into Polish.
So I thought it might be useful to others if I share it.

Happy to get any feedback, if anything would need a correction.
best
Marcin

### CODE REVIEW

File: docs/source/ultimate.rst [hunk 1]
```diff
@@ -131,15 +131,15 @@ Polish
 
 Dane i próbki z tego badania mogą zostać użyte do innych, przyszłych projektów badawczych niezależnych od badania, w którym właśnie bierzesz udział. Te przyszłe badania mogą dotyczyć dowolnego tematu, który może być niezwiązany z celami tego badania. Zapewnimy dostęp do zbieranych przez nas danych, włączając w to dane z neuroobrazowania, dla ogółu społeczeństwa za pośrednictwem internetu oraz w pełni otwartej bazy danych.
```
Comment (kfinc) [inline docs/source/ultimate.rst]: wykorzystane

File: docs/source/ultimate.rst [hunk 2]
```diff
@@ -125,6 +125,25 @@ Conserveremo i suoi dati sensibili (nome e cognome, informazioni per contattarla
...
+
+Zgodnie z naszą wiedzą, tak udostępniane ogółowi społeczeństwa dane nie będą zawierały żadnych informacji pozwalających na Twoją identyfikację. Twoje dane zostaną zanonimizowane, jednak dzięki wykorzystaniu danych powiązanych z Twoim imieniem w innych rejestrach (na przykład skanów mózgu uzyskanych z Twojej dokumentacji medycznej) istnieje możliwość powiązania Twoich dany obrazowych lub innych informacji z naszej bazy, z powrotem z Tobą. Ponadto, naruszenie zabezpieczeń (włamanie lub cyberatak) może doprowadzić do tego, że ktoś będzie w stanie zidentyfikować Twoje dane. Ryzyko wystąpienia takich zdarzeń jest jednak bardzo niskie, ponieważ Twoje dane są przechowywane w bezpiecznej bazie danych, a informacje wiążące Twoją tożsamość z nadanym kodem uczestnika badania znajdą się w innym rejestrze.
```
Comment (djarecka) [inline docs/source/ultimate.rst]: I liked the previous version "z Twoja osobą", so the sentence looks like:
`...powiązania Twoich dany obrazowych lub innych informacji z naszej bazy z Twoją osobą"

**Comment by kfinc** [comment]
Thanks, @mkoculak & @djarecka! Looks great! 
I agree with @djarecka recent suggestions and I've added one more comment.

**Comment by mkoculak** [comment]
I've inserted all the changes from the comments.
So I think it is good to go.

**Comment by yarikoptic** [comment]
I will wait for the blessing by @djarecka ;-)

**Comment by djarecka** [comment]
full blessing :-)

**Comment by yarikoptic** [comment]
THANK YOU EVERYONE!

**Pull request merged**